src/ethereum/info/arbitrumInfo.ts

```diff
@@ -0,0 +1,145 @@
+import { EdgeCurrencyInfo, EdgeTokenMap } from 'edge-core-js/types'
+
+import { makeOuterPlugin } from '../../common/innerPlugin'
+import { makeMetaTokens } from '../../common/tokenHelpers'
+import type { EthereumTools } from '../EthereumTools'
+import type { EthereumFees, EthereumNetworkInfo } from '../ethereumTypes'
+import { ethOtherMethodNames } from '../ethereumTypes'
```

Comment:
I worry about this import. It might be safer if we move `ethOtherMethodNames` into the `ethereumCommonInfo.ts` file, so we don't pull in a bunch of cleaners and other stuff from the types file.